chart/values.yaml

```
@@ -81,3 +81,9 @@ containerSecurityContext: {}
 # -- [Kubernetes deployment strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) for the deployment.
 deploymentStrategy:
   type: RollingUpdate
+
+# -- Labels added to the deployment metadata
+deploymentLabels: {}
```

```suggestion
labels: {}
```

Looking at this now, maybe we should drop the `deployment` prefix. What do you think?